source/operations/install-deploy-manage/deploy-minio-tenant.rst

```diff
@@ -565,12 +538,276 @@ the Kubernetes cluster.
 
 .. _minio-k8s-deploy-minio-tenant-commandline:
 
-Procedure (CLI)
----------------
+Procedure (Command Line)
+------------------------
 
 The :mc:`kubectl minio tenant create` command supports creating a MinIO Tenant in your Kubernetes cluster.
 The command *requires* that the cluster have a functional MinIO Operator installation.
 
-You can also use this procedure to generate a Kubernetes ``YAML``-formatted resource document for further customization.
+To deploy a tenant from the command line, complete the following steps:
+
+:ref:`create-tenant-cli-determine-settings-required-options`
+
+:ref:`create-tenant-cli-determine-additional-options`
+
+:ref:`create-tenant-cli-enter-command`
+
+:ref:`create-tenant-cli-record-access-info`
+
+:ref:`create-tenant-cli-access-tenant-console`
+
+:ref:`create-tenant-cli-forward-ports`
+
+.. _create-tenant-cli-determine-settings-required-options:
+
+1) Determine Values for Required Settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :mc:`kubectl minio tenant create` command requires several configuration settings.
+Determine the values for all required settings.
+
+.. tab-set::
+   
+   .. tab-item:: Required Settings
+
+      The command requires values for each of the following:
+
+      - :mc:`~kubectl minio tenant create TENANT_NAME`
```

Comment:
This might read cleaner as a list-table directive

```
.. list-table::
   :header-rows: 1
   :widths: 40 60
   :width: 100%

   * - Setting
      - Description

   * - ...
     - ...

```

Similar to the Operator Console sections above.

Reply:
Changed to a table